code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -65,7 +65,7 @@
 
 /obj/item/weapon/storage/briefcase/false_bottomed
 	name = "briefcase"
-	desc = "It's made of AUTHENTIC faux-leather and has a price-tag still attached. This one feels a bit heavier than normal for how much fits in it."
+	desc = "It's made of AUTHENTIC faux-leather and has a price-tag still attached. Its owner must be a real professional."
```

Comment:
You can remove this line entirely so that in the unlikely event that someone modifies the base briefcase's description, this one will automatically stay up to date